afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/events/NodeEventType.java

```diff
@@ -24,5 +24,8 @@
     BACKWARD_DEPENDENCY_REMOVED,
     TIME_SERIES_CREATED,
     TIME_SERIES_DATA_UPDATED,
-    TIME_SERIES_CLEARED
+    TIME_SERIES_CLEARED,
```

Comment:
We have here a design issue: AFS objects are designed to be added as plugin and there is many other AFS object in downstream projects (so not in powsybl core). It won't be possible to extend this enum if these AFS objects want to send an event.
We probably need to abandon this enum and use a string so keep the plugin design of AFS consistent.
@sylvlecl any opinion on this?

Reply:
I agree with you, i think that the cleanest way to resolve this problem is to abandon the enum and use a String instead.